Fix jupyter install bugs (#367)
dotnet/interactive: dotnet-interactive/CommandLine/readme.md

| Command                                                      | HTTP behavior of kernel 
|--------------------------------------------------------------|--------------------------------
dotnet interactive jupyter install                             | Disabled
dotnet interactive jupyter install --http-port-range 8001-9000 | Enabled on autoselected port between 8001 and 9000                      

 

## Installing kernels

> When installing `dotnet-interactive` it is possible to provide the path for installing the kernel specs, the path msut exists.

Reviewer: ```suggestion
When installing `dotnet-interactive` it is possible to provide the path for installing the kernel specs. The provided path must be an existing directory.
```

| Command                                                      | Destination path
|--------------------------------------------------------------|--------------------------------

> dotnet interactive jupyter install                             | Will use the kernelspec module if available, if it is not present will attempt ot use platform specific folders for python or anaconda paths

Reviewer: ```suggestion
dotnet interactive jupyter install                             | Use the kernelspec module if available. If not found, attempt to use well-known platform-specific folders for Python or Anaconda .
```

Author: don
